pull-request.sh

```diff
@@ -106,6 +120,15 @@ main () {
         PULL_REQUEST_DRAFT="true"
     fi
 
+    if [ -z "${PULL_REQUEST_MAINTAINER_CAN_MODIFY}" ]; then
+        echo "No explicit preference for maintainers can modify PR: created PRs with maintainers can modify=true."
+        PULL_REQUEST_MAINTAINER_CAN_MODIFY="true"
+    else
+        echo "Environment variable PULL_REQUEST_MAINTAINER_CAN_MODIFY set to a value: created PRs with maintainers can modify=false."
+        PULL_REQUEST_MAINTAINER_CAN_MODIFY="false"
```

Comment:
Yes, well done here! Here you are giving the user feedback that the variable is set (although they probably already knew this since they set it).